extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -54,12 +53,36 @@ amp-story-audio-sticker {
   width: 100% !important;
   max-width: 100% !important; /* Set the maximum width of the image to 100% to ensure it fits within the container */
   max-height: 100% !important; /* Set the maximum height of the image to 100% to ensure it fits within the container */
+
+  --story-audio-sticker-outline-color: white !important;
+  --story-audio-sticker-dropshadow-color: white !important;
 }
 
 amp-story[muted] amp-story-audio-sticker {
   pointer-events: auto !important;
 }
 
+.i-amphtml-amp-story-audio-sticker-container.large amp-story-audio-sticker {
+  border-radius: 40px !important;
+}
+
+.i-amphtml-amp-story-audio-sticker-container.small amp-story-audio-sticker {
+  border-radius: 25px !important;
+}
+
+amp-story-audio-sticker[sticker-style="outline"] {
+  box-shadow: inset 0 0 0 4px var(--story-audio-sticker-outline-color) !important;
+}
+
+amp-story-audio-sticker[sticker-style="dropshadow"] {
+  box-shadow: 4px 4px 20px var(--story-audio-sticker-dropshadow-color) !important;
+}
+
+amp-story[muted] amp-story-audio-sticker:active {
```

Comment:
Keep the transition even if it's not muted, so it can transition smoothly after you click on it (now it snaps back to the original size).

```css
audio-sticker {
  transition: ...;
}
amp-story[muted] audio-sticker:active {
  transform: ...
}
```

Reply:
Done.